libs/features/mas/src/price/template.js

```diff
@@ -136,11 +137,10 @@ function renderContainer(
         taxInclusivityLabel,
         true,
     );
-
-    return renderSpan(cssClass, markup, {
+    const regularlySRLabel = accessibleLabel ? `<sr-only class="strikethrough-aria-label">${accessibleLabel}</sr-only>` : '';
```

Comment:
can we instead do at the line 127:
let markup = accessibleLabel ? `<sr-only class="strikethrough-aria-label">${accessibleLabel}</sr-only>` : '';